docs/UsersGuide/source/Graphics.rst

```diff
@@ -151,12 +99,15 @@ is as follows:
 
 .. code-block:: console
 
-   python plot_allvars.py 2019061500 0 12 6 /path-to/expt_dirs/test_CONUS_25km_GFSv16 /path-to/NaturalEarth RRFS_CONUS_25km
+   python plot_allvars.py 2019061518 0 12 6 /path-to/expt_dirs/test_community /path-to/NaturalEarth RRFS_CONUS_25km
 
 The output files (in ``.png`` format) will be located in the directory ``$EXPTDIR/CDATE/postprd``,
 where in this case ``$EXPTDIR`` is ``/path-to/expt_dirs/test_CONUS_25km_GFSv16`` and ``$CDATE`` 
 is ``2019061518``.
 
+.. note::
+   The plotting script may generate a series of deprecation warnings, but users can safely ignore these. The script will generate the plots successfully despite them. 
+
```

Comment:
I have suppressed all depreciation and other warnings in the plotting scripts in PR #481. Also they are now part of the workflow so their outputs go to the log file.

Reply:
@danielabdi-noaa Thanks for doing that! Out of curiosity, is any work being done to update the plotting environment so that deprecation warnings don't need to be suppressed (because all the packages are recent enough)? 
@JeffBeck-NOAA We talked about doing looking into avoiding/muting the warnings for the release, but since we were already in code freeze, it seemed unwise to add any additional code (a) because it's not necessary/not a bug (the code still works) and (b) because it wasn't clear how long the changes would take. We were planning to look at it for the next release, but it looks like Daniel got a head start on us! ;)

Reply:
@gspetro-NOAA, sounds great, thanks!

Reply:
@gspetro-NOAA Some depreciation warnings were in the scripts themselves and were easy to fix, however, some of the packages were still spitting out depreciation warnings. I think this is often the case when you have so many dependencies in a python app, so I wouldn't worry about it too much.

I think I found a few bugs in `plot_diff.py` scripts which maybe critical enough to make a PR to the release branch. One is  there seems to be a mismatch in the number of tick labels and number of ticks, so I was not able to successfully run the diff script because of it for the specific test case I was running. 
```
The number of FixedLocator locations (18), usually from a call to set_ticks, does not match the number of ticklabels
```
Now that it is part of the workflow, it is exposed to lots of WE2E test cases. Another issue I found was that some runs may not have 500mb variables in the grib2 files, so I added try-except caluse to fix that. I am not familiar with the scripts so maybe  @BenjaminBlake-NOAA  (developer of the script) can take a look at those